custom_components/babybuddy/services.yaml

```diff
@@ -147,143 +93,184 @@ services_feedings_add:
       name: Feeding amount
       description: Feeding amount in number format
       selector:
-        text:
+        number:
+          min: 1
+          max: 300
+          mode: box
     notes:
       name: Feeding notes
       description: Feeding notes
       selector:
         text:
+          multiline: true
 
-services_notes_add:
-  name: Add Note
-  description: Adds a note
+add_sleep:
+  name: Add Sleep
+  description: Adds sleep entry
+  target:
+    entity:
+      integration: babybuddy
+      domain: switch
   fields:
-    child:
-      name: Child
-      description: Child to add a note to
-      required: true
+    timer:
+      name: Use timer
+      description: Use active timer to assign child/start/end
       selector:
-        entity:
-    note:
-      name: Note
-      description: Note
-      required: true
+        boolean:
+    start:
+      name: Sleep start time
+      description: Sleep start time
       selector:
-        text:
-    time:
-      name: Note time
-      description: Note time in YYYY-MM-DD HH:MM format
+        time:
+    end:
+      name: Sleep end time
+      description: Sleep end time
+      selector:
+        time:
+    notes:
+      name: Sleep notes
+      description: Sleep notes
       selector:
         text:
+          multiline: true
 
-services_sleep_add:
-  name: Add Sleep
-  description: Adds sleep
+add_tummy_time:
+  name: Add Tummy Time
+  description: Adds Tummy Time entry
+  target:
+    entity:
+      integration: babybuddy
+      domain: switch
```

Comment:
should this be switch?

Reply:
`add_tummy_time` service is part of the switch entity because a feeding can be linked to a timer (which is a switch in HA). If timer option is enabled in the service call the timer ID is used in the timer field in the api call. This allows to automatically assign the start/end times based on the active timer.